aten/src/ATen/core/TensorImpl.h

```diff
@@ -114,7 +247,8 @@ struct CAFFE2_API TensorImpl : public c10::intrusive_ptr_target {
         ", while tensor contains ",
         data_type_.name(),
         ". ");
-    return storage_.data<T>() + storage_offset_;
+    // We managed the type check ourselves
+    return storage_.unsafe_data<T>() + storage_offset_;
   }
 
   inline void* data() const {
```

Comment:
const is dropped from c2 data() API, is it OK?

Reply:
Yeah, it is not strictly backwards compatible, but I only actually needed to fix, like, three call sites.